### ISSUE TITLE
Add Project Euler solution which are not there in this repository - C/C++ Language only

### ISSUE THREAD

**Issue by amanBhawsar** [amanBhawsar/Project-Euler#1]
Guideline for naming files => 
Project Euler #{Question number} {Question name as on HackerRank}.{c/c++}
Please add solutions accepted on ProjectEuler on hackerRank platform.

Cheers

**Comment by maanpatel2492** [comment]
We can give solutions to any problem on  ProjectEuler or on Hackerrank platform right?

**Comment by amanBhawsar** [comment]
Yes @maanpatel2492  which are not currently present in repo. Raise PR of 5+ Questions at once.
